parquet-column/src/test/java/org/apache/parquet/schema/TestMessageType.java

```diff
@@ -145,4 +145,18 @@ public void testIDs() throws Exception {
     assertEquals(schema, schema2);
     assertEquals(schema.toString(), schema2.toString());
   }
+
+  @Test
+  public void testEmptyGroup() {
+    try {
+      MessageType t5 = new MessageType("root1",
+          new GroupType(REQUIRED, "g1"),
+          new GroupType(REQUIRED, "g2",
+              new PrimitiveType(OPTIONAL, BINARY, "b")));
```

Comment:
you need to put:
`fail("this should throw")` 
in this try block -- if this never throws the test will still pass